### ISSUE TITLE
DUP Accessibility issues when booking and cancelling a pass

### ISSUE THREAD

**Issue by kgorana** [bcgov/parks-reso-public#283]
### Issue: When using text to speech/screen reader opening the `Pass Type` dropdown does not automatically read the first option.
![Image](https://github.com/bcgov/parks-reso-public/assets/145403218/2e34872e-682c-47ef-8e6b-6ca428bc4a8a)

**How to replicate:**
* Using a screen reader open the dropdown for `Pass Type` either with keyboard or mouse.

**Expected Behavior:**
* First option should be read out loud without having to be selected/highlighted first
___
### Issue: VoiceOver reads the days of the week as the abbreviated versions shown at the top of the calendar.
![Image](https://github.com/bcgov/parks-reso-public/assets/145403218/3709093b-9d7a-423b-b614-b06c978c45b3)
**How to replicate:**
* Using a VoiceOver for macOS/IOS open the `Visit Date` calendar
* Days of the week are read out as the abbreviated forms shown in the calendar

**Expected Behavior:**
* Full day name should be read as it is with other screen readers

> [!Note]
This could be a limitation of the VoiceOver software and needs some investigation to see if it is even possible to fix. 
___
### Issue: Most screen readers read the name of icons, so the arrow before the `Home` link is read as _arrow underscore back_. The whole thing is read _arrow underscore back home button_. This is true for any button with this format: 
![Image](https://github.com/bcgov/parks-reso-public/assets/145403218/29b95012-2027-4540-b04e-6c7d3dcc9215)

![Image](https://github.com/bcgov/parks-reso-public/assets/145403218/dd3f7588-4d7f-4922-9993-6f84b7fb8f70)
